pep/package.json

```diff
@@ -65,7 +64,6 @@
         "babel-eslint": "^10.1.0",
         "bootstrap": "4.5.1",
         "broccoli-asset-rev": "^3.0.0",
-        "color": "^3.1.3",
```

Comment:
@bakerac4 did you use `yarn remove` to remove this dep? doesnt look like there were any changes to the yarn.lock in this push.